support/db/user.toml

```diff
@@ -1,3 +1,3 @@
-listen_addresses = "127.0.0.1"
-search_path = "public"
-hba_file = "/hab/svc/postgresql/pg_hba.conf"
```

Comment:
Its been a while since i've used PG but do we really want to make access via username and password without a pg_hba.conf? Feels like checking that into a repo could be... dangerous